assets/js/modules/adsense/settings/adsense-settings.js

```diff
@@ -179,11 +210,50 @@ class AdSenseSettings extends Component {
 							}
 						</Fragment>
 						: <Fragment>
-							{	__( 'The AdSense code has', 'google-site-kit' ) } {
-								useSnippet
-									? __( 'been placed on your site.', 'google-site-kit' )
-									: __( 'not been placed on your site.', 'google-site-kit' )
-							}
+							<div className="googlesitekit-settings-module__meta-items">
+								<div className="googlesitekit-settings-module__meta-item">
+									<p className="googlesitekit-settings-module__meta-item-type">
+										{ __( 'Publisher ID', 'google-site-kit' ) }
+									</p>
+									<h5 className="googlesitekit-settings-module__meta-item-data">
+										{ accountID }
+									</h5>
+								</div>
+								<div className="googlesitekit-settings-module__meta-item">
+									<p className="googlesitekit-settings-module__meta-item-type">
+										{ __( 'AdSense Code', 'google-site-kit' ) }
+									</p>
+									<h5 className="googlesitekit-settings-module__meta-item-data">
+										{	__( 'The AdSense code has', 'google-site-kit' ) } {
```

Comment:
This approach to translation doesn't really work, because it spilts up the i18n string into two sections that might not make sense in certain languages. This `<h5>` should change to:

```js
{ useSnippet && __( 'The AdSense code has been placed on your site.', 'google-site-kit' ) }
{ ! useSnippet && __( 'The AdSense code has not been placed on your site.', 'google-site-kit' ) }
```